script/upgrades/eXOF/eXOFChecks.verify.sol

```diff
@@ -0,0 +1,504 @@
+      );
+      // verify asset1 is always a collateral asset
+      require(
+        pool.asset1 == celoToken || pool.asset1 == bridgedUSDC || pool.asset1 == bridgedEUROC,
+        "asset1 is not CELO or bridgedUSDC in the exchange"
+      );
+    }
+    console.log("🟢 PoolExchange correctly configured 🤘🏼");
+  }
+
+  function verifyPoolConfig(eXOFConfig.eXOF memory config) internal view {
+    for (uint256 i = 0; i < config.pools.length; i++) {
+      bytes32 exchangeId = getExchangeId(config.pools[i].asset0, config.pools[i].asset1, config.pools[i].isConstantSum);
+      IBiPoolManager.PoolExchange memory pool = BiPoolManager(biPoolManagerProxy).getPoolExchange(exchangeId);
+      Config.Pool memory poolConfig = config.pools[i];
+
+      if (pool.config.spread.unwrap() != poolConfig.spread.unwrap()) {
+        console.log(
+          "The spread of deployed pool: %s does not match the expected spread: %s.",
+          pool.config.spread.unwrap(),
+          poolConfig.spread.unwrap()
+        );
+        revert("spread of pool does not match the expected spread. See logs.");
+      }
+
+      if (pool.config.referenceRateFeedID != poolConfig.referenceRateFeedID) {
+        console.log(
+          "The referenceRateFeedID of deployed pool: %s does not match the expected referenceRateFeedID: %s.",
+          pool.config.referenceRateFeedID,
+          poolConfig.referenceRateFeedID
+        );
+        revert("referenceRateFeedID of pool does not match the expected referenceRateFeedID. See logs.");
+      }
+
+      if (pool.config.minimumReports != poolConfig.minimumReports) {
+        console.log(
+          "The minimumReports of deployed pool: %s does not match the expected minimumReports: %s.",
+          pool.config.minimumReports,
+          poolConfig.minimumReports
+        );
+        revert("minimumReports of pool does not match the expected minimumReports. See logs.");
+      }
+
+      if (pool.config.referenceRateResetFrequency != poolConfig.referenceRateResetFrequency) {
+        console.log(
+          "The referenceRateResetFrequency of deployed pool: %s does not match the expected: %s.",
+          pool.config.referenceRateResetFrequency,
+          poolConfig.referenceRateResetFrequency
+        );
+        revert(
+          "referenceRateResetFrequency of pool does not match the expected referenceRateResetFrequency. See logs."
+        );
+      }
+
+      if (pool.config.stablePoolResetSize != poolConfig.stablePoolResetSize) {
+        console.log(
+          "The stablePoolResetSize of deployed pool: %s does not match the expected stablePoolResetSize: %s.",
+          pool.config.stablePoolResetSize,
+          poolConfig.stablePoolResetSize
+        );
+        revert("stablePoolResetSize of pool does not match the expected stablePoolResetSize. See logs.");
+      }
+    }
+    console.log("🟢 Pool config is correctly configured 🤘🏼");
+  }
+
+  function verifyTradingLimits(eXOFConfig.eXOF memory config) internal view {
```

Comment:
I think we are just verifying trading limits on asset 0 here since we introduce an exchange with limits configured for both assets in this deployment I would add a check for asset1 trading limits as well